stepzen-dev/stepzen-vercel-starter-template
# prepare to be the default StepZen template on Vercel

- update README
- make use of StepZen GraphQL API in the app

How to test (after the PR is merged):
 - click the "Deploy to Vercel" button
 - follow the process on Vercel and StepZen dashboard
 - see that the app gets deployed
 - clone the repo locally
 - follow the steps in README
 - see that the app starts locally
